src/main/java/pl/betoncraft/betonquest/Instruction.java

```diff
@@ -145,7 +145,7 @@ public VariableNumber getVarNum(final String string) throws InstructionParseExce
         }
         try {
             return new VariableNumber(pack.getName(), string);
-        } catch (NumberFormatException e) {
+        } catch (InstructionParseException e) {
```

Comment:
Doesn't this just throw a NumberFormatException instead of the InstructionParseException?